src/mappingForLepton.ts

```diff
@@ -85,6 +85,14 @@ export function handleLeptonBatchMinted(event: LeptonBatchMinted): void {
   //
   // WARNING: Event Param: tokenId is INCORRECT in this event - do not rely on it!
   //
+  let startTokenId = event.params.tokenId;
```

Comment:
As per the comment `event.params.tokenId` is incorrect, so this whole code block is invalid.
Please remove